### PR TITLE
Add Swager UI

### LINKED ISSUE #4: Setup Swagger
Blocked by #1 

### What Functionality Are You Suggesting?
We want to set up a swagger document that automatically contains all the documentation for the API. This functionality is described on the NestJS documentation [here](https://docs.nestjs.com/openapi/introduction).

### Why Will This Benefit The Community?
This will allow developers who are working on the frontend to easily see how to save and retrieve data to and from the API.

### PR DESCRIPTION
**Pull request by JonaVDM**
Resolves #4

### Overview
- Installed swagger

### CODE REVIEW

File: src/app.controller.ts [hunk 1]
```diff
@@ -1,11 +1,14 @@
 import { Controller, Get } from '@nestjs/common';
 import { AppService } from './app.service';
+import { ApiResponse, ApiTags } from "@nestjs/swagger";
```
Comment (LamboCreeper) [inline src/app.controller.ts:3]: Can this import be above the local one? Keep all imports from external packages at the top.

File: src/main.ts [hunk 2]
```diff
@@ -1,12 +1,24 @@
 import 'dotenv/config';
 import { NestFactory } from '@nestjs/core';
 import { AppModule } from './app.module';
+import { DocumentBuilder, SwaggerModule } from '@nestjs/swagger';
```
Comment (LamboCreeper) [inline src/main.ts:4]: Same here.

**Review by LamboCreeper — CHANGES_REQUESTED** [review]

File: src/app.controller.ts (hunk 1 above)
Comment (JonaVDM) [reply src/app.controller.ts:2]: Done

File: src/main.ts (hunk 2 above)
Comment (JonaVDM) [reply src/main.ts:3]: Done

**Review by LamboCreeper — APPROVED** [review]
Thanks Jona!

**Pull request merged**